gitpod-io/website
# ✨ Wording changes to `/events` & `/guides`

## Description

- Text Copy changes to  `/events` & `/guides` page

## Related Issue(s)



Fixes #3673

## How to test



- Open preview
- Go to  `/events` & `/guides`
- See :eyes:

## Release QA Checklist ✔️



<details>
  <summary>QA Checklist </summary>

## [Priority: High] Functional Testing

  

## [Priority: High] Content Testing


## [Priority: Medium] Performance Testing


## [Priority: Medium] Accessibility Testing


By including these key items in your website release QA checklist, you can ensure that your website is fully functional, user-friendly, and meets your business requirements.

## Security Testing

> **Note**:💡 If we are changing something related to cookies or data storing or analytics side.


  </details>
